### PR TITLE
k8s: add Kubernetes Dashboard workflow

### PR DESCRIPTION
**Pull request by rithujohn191**
### Description
This PR adds a new workflow for the Kubernetes Dashboard which serves as an overview of all the Kubernetes resources running in the cluster. This PR adds a Pods list table which displays pods in a given namespace in a clientset and allows the user to perform a delete operation on a pod from the dashboard

### Testing Performed
Local 
ezgif com-gif-maker (48)

### TODOs
- Add more resource tables to this dashboard. Tables for Deployments, CronJobs, StatefulSets and Jobs coming soon!

### CODE REVIEW

File: frontend/workflows/k8s/package.json [hunk 1]
```diff
@@ -26,9 +26,14 @@
...
     "@clutch-sh/data-layout": "^1.0.0-beta",
     "@clutch-sh/wizard": "^1.0.0-beta",
+    "@emotion/styled": "^11.1.5",
+    "@hookform/resolvers": "2.0.0-beta.3",
+    "@material-ui/icons": "^4.9.1",
     "lodash": "4.17.21",
     "react": "^16.8.0",
     "react-dom": "^16.8.0",
+    "react-hook-form": "^6.11.0",
+	"react-router-dom": "^6.0.0-beta",
```
Comment (smonero) [inline frontend/workflows/k8s/package.json:36]: nit: spacing

**Review by smonero — APPROVED** [review]

File: frontend/workflows/k8s/package.json (hunk 1 above)
Comment (rithujohn191) [reply frontend/workflows/k8s/package.json]: thanks for catching that

**Comment by scarlettperry** [comment]
a quick comment - i think the dashboard files should be grouped in a K8s dashboard folder since its made up of a few components

**Comment by rithujohn191** [comment]
> a quick comment - i think the dashboard files should be grouped in a K8s dashboard folder since its made up of a few components

I was under the impression that the index.tsx file needs to bee at the same level as the other component tsx files. If i try to put it in a separate directory it complains about the hierarchy structure.

**Review by smonero — APPROVED** [review]

**Comment by rithujohn191** [comment]
addresses : #520

**Pull request merged**